subsys/fb/Kconfig

```diff
@@ -31,6 +38,7 @@ if CHARACTER_FRAMEBUFFER_SHELL
 	string
 	default "SSD1673" if SSD1673
 	default "SSD1306" if SSD1306
+	default "ELCDIF_1" if DISPLAY_MCUX_ELCDIF
```

Comment:
SSD1306 and SSD1673 is also put here. I just follow the default ones.

Reply:
Reasoning behind the question is that board Kconfig.defconfig already contains one for LVGL and I thought it is better to group them in one place

Reply:
so move the  CHARACTER_FRAMEBUFFER_SHELL to board? this may need to change other platform which may not owned by me. can we just keep it as is for this time of being? @vanwinkeljan

Reply:
I dont see how moving a config in to a board specific defconfig file would impact another platform.
The same has been done for the LVGL config for the same board:
https://github.com/zephyrproject-rtos/zephyr/blob/e0943cfaab29100c5212f74aa9f3eb1a4aa58f0b/boards/arm/mimxrt1050_evk/Kconfig.defconfig#L76

Reply:
LVGL_DISPLAY_DEV_NAME  in board default config, as the lvgl sample needs a display name, so maureen name it LVGL_DISPLAY_DEV_NAME, but in cfb cases the original code using the DISPLAY_NAME to refer to the esiplay driver. So if we keep adding defines in this way, the default board config will be too big. So the root cause is that every application developer create their own display naming, for cfb_shell it is  CHARACTER_FRAMEBUFFER_SHELL, for cfb it is the DISPLAY_NAME, and for lvgl it is LVGL_DISPLAY_DEV_NAME. I guess we can keep the rules as is before we can get aligned, any commnets? @vanwinkeljan

Reply:
> So if we keep adding defines in this way, the default board config will be too big.

Actually  the defines could go into the board specific conf files of the samples.
So maybe better move it over there if the concern is that the board default Kconfig will grow to large.

>So the root cause is that every application developer create their own display naming

Main reason for this is the you could have multiple displays and both LVGL and CFB shell init run during system startup.

Reply:
ok， let me create the board config for each samples